### PR TITLE
Stateful copy paste

### PR DESCRIPTION
**Pull request by mmcdermott**

### CODE REVIEW

File: src/modules/copied/actions.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export function paste(spaceId){
+  return (dispatch, getState) => {
+    const state = getState()
+    if (!(state.copied && state.selection)) { return }
+
+    const {metric, guesstimate} = state.copied
+
+    const location = state.selection
+    const spaceMetrics = getState().metrics.filter(m => m.space === spaceId)
+    const existentMetric = spaceMetrics.find(m => isAtLocation(m, location))
```
Comment (OAGr) [inline src/modules/copied/actions.js]: existingMetric ?

File: src/components/lib/FlowGrid/FlowGrid.jsx [hunk 2]
```diff
@@ -45,13 +47,26 @@ export default class FlowGrid extends Component{
 
   state = { rowHeights: [] }
 
-  _handleKeyPress(e) {
+  _handleKeyUp(e){
+    if (e.keyCode == '17') {
+      this.setState({ctrlPressed: false})
```
Comment (OAGr) [inline src/components/lib/FlowGrid/FlowGrid.jsx]: also do for cmd for macs :)

File: src/modules/copied/actions.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    const existingReadableIds = spaceMetrics.map(m => m.readableId)
+
+    let newMetric = e.metric.create(existingReadableIds)
+    newMetric.location = location
+    const newItem = Object.assign({}, metric, newMetric)
```
Comment (OAGr) [inline src/modules/copied/actions.js]: I'd write these three lines above as, if possible,

``const newItem = {...metric, ...newMetric, location}``

Comment (mmcdermott) [inline src/modules/copied/actions.js]: That doesn't appear to work. Perhaps I'm misunderstanding, but as written that doesn't appear to work.

**Pull request merged**